src/codegen/codegen_c_visitor.cpp

```diff
@@ -2656,11 +2656,13 @@ void CodegenCVisitor::print_mechanism_register() {
             method_name("net_buf_receive")));
     }
     if (info.num_net_receive_parameters != 0) {
-        printer->add_line("pnt_receive[mech_type] = {};"_format(method_name("net_receive")));
-        printer->add_line("pnt_receive_size[mech_type] = num_net_receive_args();");
+        auto net_recv_init_arg = "NULL";
```

Comment:
From my understanding `net_recv_init_arg/pnt_receive_init` is a pointer to a function name so it could also be like that?
```suggestion
        auto net_recv_init_arg = "nullptr";
```